string node
---
string[] service_list